.github/workflows/create-release.yml

```diff
@@ -35,3 +37,197 @@ jobs:
+          restore-keys: |
+            ${{ runner.os }}-maven-
+
+      - name: Cache Built Code
+        uses: actions/cache@v2
+        with:
+          key: built-code-${{ github.run_id }}
+          path: ./**/target
+
+      - name: Set up JDK
+        uses: actions/setup-java@v2
+        with:
+          java-version: '8'
+          distribution: 'adopt'
+
+      - name: Configure git user
+        run: |
+          git config user.name "liquibot"
+          git config user.email "liquibot@liquibase.org"
+
+      - name: Download and install liquibase.jar
+        uses: dsaltares/fetch-gh-release-asset@master
+        with:
+          repo: "liquibase/liquibase"
+          version: "tags/v${{ needs.setup.outputs.liquibaseVersion }}"
+          file: "liquibase-${{ needs.setup.outputs.liquibaseVersion }}.jar"
+          target: "liquibase.jar"
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Install liquibase.jar
+        run: mvn -B org.apache.maven.plugins:maven-install-plugin:3.0.0-M1:install-file -Dfile=liquibase.jar
+
+      - name: Update pom.xml with release versions and commit changes
+        run: |
+          mvn -B versions:set -DnewVersion=${{ needs.setup.outputs.extensionVersion }} -DallowSnapshots=false -DoldVersion="*"
+          mvn -B versions:use-dep-version -Dincludes=org.liquibase:liquibase-core -DdepVersion=${{ needs.setup.outputs.liquibaseVersion }} -DforceVersion=true
+
+          git add pom.xml
+          git commit -m "Version Bumped to ${{ needs.setup.outputs.extensionVersion }}"
+          git tag -a -m "Version Bumped to ${{ needs.setup.outputs.extensionVersion }}" liquibase-postgresql-${{ needs.setup.outputs.extensionVersion }}
+          git push "https://liquibot:$GITHUB_TOKEN@github.com/$GITHUB_REPOSITORY.git" HEAD:${{ github.ref }} --follow-tags --tags
+        env:
+          GITHUB_TOKEN: ${{ secrets.BOT_TOKEN }}
+
+      - name: Get release SHA
+        id: get-release-sha
+        run: echo ::set-output name=releaseSha::$(git rev-parse HEAD)
+
+
+      - name: Build and Unit Test
+        run: mvn -B clean test package
+
+      - name: Archive Test Results
+        if: ${{ always() }}
+        uses: actions/upload-artifact@v2
+        with:
+          name: test-reports-jdk
+          path: ./**/target/surefire-reports
+
+      - name: Save Artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: liquibase-postgresql
+          path: |
+            target/*.jar
+
+  integration-tests:
+    name: Java ${{ matrix.java }}, Postgres ${{ matrix.postgres }}
+    runs-on: ubuntu-latest
+    needs: build
+    strategy:
+      matrix:
+        java: [8]
+        postgres: [9, 10, 11, 12, 13]
+    services:
+      postgres:
+        image: postgres:${{ matrix.postgres }}
+        env:
+          POSTGRES_USER: lbtest
+          POSTGRES_PASSWORD: lbtest
+        ports:
+          - 5432:5432
+    steps:
+      - uses: actions/checkout@v2
+
```

Comment:
Should there also be a download artifact step here to test the previously built artifact?

Reply:
In theory, yes. This was the code from before and I was trying to only change what I had to. It shows as new because it was in a different file before.

In the liquibase/liquibase repo I've been having the integration tests use the build from the previous step so I have an example I can pull from if we'd like to change that now vs. in a future improvement.

Reply:
Actually, I already am using the previously built artifact. The "Cache Built Code" part should load in the previously built code for use in the tests